<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JournalCreator</class>
 <widget class="QDialog" name="JournalCreator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Journal Creator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/icons/database_add.png</normaloff>:/icons/database_add.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="TabBox">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="Type">
      <attribute name="title">
       <string>1. Journal Type</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="margin">
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Choose new journal database type</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="DBType">
         <item>
          <property name="text">
           <string>SQLite (recommended)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>MySQL (advanced users)</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="Description"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="BasicSettings">
      <attribute name="title">
       <string>2. Basic Settings</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <property name="margin">
        <number>6</number>
       </property>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Define the essential parameters of your new journal:</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Journal name</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="JournalName">
         <property name="toolTip">
          <string>This is the name that will identify this journal on the host.</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Journal host</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="Hostname">
         <property name="toolTip">
          <string>This is the machine on which the journal will be created. If the field is left blank the journal is created on this computer.</string>
         </property>
         <property name="placeholderText">
          <string>localhost</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Username</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="Username">
         <property name="toolTip">
          <string>This is the user that will be associated with this journal. If User already exists, this journal will be added to the group that this user controls. </string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Password</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="Password">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Repeat</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="Password2">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLabel" name="MatchNotify">
         <property name="text">
          <string>Passwords do not match</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0" colspan="2">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Preferred</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>4</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Authentication">
      <attribute name="title">
       <string>3. Authentication</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="margin">
        <number>6</number>
       </property>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>&lt;p&gt;RoboJournal requires root access to your MySQL installation to create the database and set up your new user account. Without the correct root password, the journal creation process will fail.&lt;/p&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Root password</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="rootPassword">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Port</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="Port"/>
       </item>
       <item row="5" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>27</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="UseAsDefault">
     <property name="toolTip">
      <string>&lt;P&gt;If checked, this setting will instruct RoboJournal's configuration process to use this journal as the default. If this is the first time you have used RoboJournal, this option will be required since there are no other journals to use in that situation. &lt;/p&gt;</string>
     </property>
     <property name="text">
      <string>Set this new journal as my default</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>4</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
